polybar-scripts/wakatime-status/polybar_wakatime.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+TOKEN="$(echo MYTOKEN | base64)"
+curl -s --header "Authorization: Basic ${TOKEN}"  https://wakatime.com/api/v1/users/current/status_bar/today  | jq -r '.data.grand_total.text';
```

Comment:
You probably have checked `curl -u` here right?
I guess it wont work, so your workaround seems to be the best solution.